src/game/boe.main.cpp

```diff
@@ -387,19 +387,17 @@ using Key = sf::Keyboard::Key;
 std::map<Key,int> delayed_keys;
 const int ARROW_SIMUL_FRAMES = 3;
 
-// Terrain map coordinates to simulate a click for 8-directional movement/waiting
-const int SOUTHWEST = 1;
-const int SOUTH = 2;
-const int SOUTHEAST = 3;
-const int WEST = 4;
-const int EAST = 6;
-const int NORTHWEST = 7;
-const int NORTH = 8;
-const int NORTHEAST = 9;
+// Terrain map coordinates to simulate a click for 8-directional movement/waiting,
+// ordered to correspond with eDirection
 location terrain_click[10] = {
-	{150,185},{120,215},{150,215},{180,215},
-	{120,185},{150,185},{180,185},
-	{120,155},{150,155},{180,135}
+	{150,155}, // north
+	{180,135}, // northeast
+	{180,185}, // east
+	{180,215}, // southeast
+	{150,215}, // south
+	{120,215}, // southwest
+	{120,185}, // west
+	{120,155}, // northwest
```

Comment:
I was momentarily confused by the fact that there are now 8 entries instead of 10, but looking at the code below, it makes sense – the other version has 2 identical entries for waiting (numpad 0 or 5), but this one doesn't handle waiting at all.

But in that case, I think the comment and the array size should be changed.